src/common/debugSession.ts

```diff
@@ -327,57 +362,51 @@ export class AvmDebugSession extends DebugSession {
     args: DebugProtocol.StackTraceArguments,
   ): void {
     try {
-      const startFrame =
-        typeof args.startFrame === 'number' ? args.startFrame : 0;
-      const maxLevels = typeof args.levels === 'number' ? args.levels : 1000;
-
-      // The runtime has a stack where the latest call is the last element. We need to return the
-      // reverse of that.
-      const adjustedEndFrame = this._runtime.stackLength() - startFrame;
-      const adjustedStartFrame = Math.max(0, adjustedEndFrame - maxLevels);
+      const startFrame = args.startFrame || 0;
+      const maxLevels = args.levels || 1000;
 
-      const stk = this._runtime.stack(adjustedStartFrame, adjustedEndFrame);
+      const stk = this._runtime.stack(startFrame, startFrame + maxLevels);
```

Comment:
Assuming this was done on purpose?

Reply:
See https://github.com/algorandfoundation/avm-debugger/pull/2/files/e5546a104221e452aa4a4916665609efb04a6a28#diff-57714ab2de4315a6d3e2e1e17241bd601aabb7f0c09bd26debf698925f8493d4R224